File: 권순우/LV2/최솟값 만들기.py
#1
def solution(A,B):
    res = 0
    A.sort()
    B.sort(reverse=True)
    for i in range(len(A)):
        res = res + A[i]*B[i]
    return res

#2
def solution(A,B):
    A.sort()
    B.sort(reverse=True)
    
    res = 0
    for i in range(len(A)):
        res += A[i] * B[i]
        
    return res

File: 조은정/Lv.2/최솟값 만들기.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> A, vector<int> B)
{
    int answer = 0, sz = A.size();
    sort(A.begin(), A.end());
    sort(B.begin(), B.end());
    for (int i = 0; i < sz; ++i)
        answer += A[i] * B[sz - 1 - i];
    return answer;
}